### DISCUSSION TITLE
Setting up on Mac client

### DISCUSSION THREAD

**Discussion by daemontroll** [sickcodes/Docker-OSX#399] [General]
I am running docker on my Mac and trying to start the container but when I do I get an error about the /dev/kvm.  Anyone see this?  Full error below:

error gathering device information while adding custom device "/dev/kvm": no such file or directory.
ERRO[0001] error waiting for container: context canceled

**Comment by PaulSender** [comment]
Getting the same, I think it's an issue with the M1 chip. 

WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
docker: Error response from daemon: error gathering device information while adding custom device "/dev/kvm": no such file or directory.

see warning.

**Comment by pcgandhinj466** [comment]
> I am running docker on my Mac and trying to start the container but when I do I get an error about the /dev/kvm.  Anyone see this?  Full error below:
> 
> 
> 
> error gathering device information while adding custom device "/dev/kvm": no such file or directory.
> 
> ERRO[0001] error waiting for container: context canceled
